src/pages/Map.js

```diff
@@ -19,7 +26,26 @@ export default function Map() {
         }}
         defaultCenter={location}
         defaultZoom={zoomLevel}
-      ></GoogleMapReact>
+        // onGoogleApiLoaded={({ map, maps }) => renderMarkers(map, maps, locations)}
+        yesIWantToUseGoogleMapApiInternals
+      >
+        {locations &&
+          locations.map((locationData) => {
+            console.log(locationData.thumbnail);
+            return (
+              <LocationPin
+                img={
```

Comment:
![image](https://user-images.githubusercontent.com/44718574/104820517-f4251700-57e9-11eb-91b7-52a686f26a7b.png)
This is the image I saw when it looks like there is no thumbnail maybe? The AC says"Pins should render a default pin image when no thumbnail is returned".